Swift Code Challenge/Views/Components/UsersListComponent.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UsersListComponent.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 18/07/22.
+//
+
+import SwiftUI
+
+struct UsersListComponent: View {
```

Comment:
This should be `UserListView` since that's actually what it does. 

```suggestion
struct UserListView: View {
```